src/lib/shell/BUILD.gn

```diff
@@ -112,6 +112,11 @@ static_library("shell") {
       "MainLoopDefault.cpp",
       "streamer_openiotsdk.cpp",
     ]
+  } else if (chip_device_platform == "asr") {
```

Comment:
Do not change file pemissions of files. Files should not be executable (do not use 755, 644 was correct)